diff -rupw openssh-9.1p1.orig/pathnames.h openssh-9.1p1/pathnames.h
--- openssh-9.1p1.orig/pathnames.h	2023-01-10 18:23:18.296391295 +0800
+++ openssh-9.1p1/pathnames.h	2023-01-10 18:36:50.298349154 +0800
@@ -41,7 +41,7 @@
 #define _PATH_HOST_ED25519_KEY_FILE	SSHDIR "/ssh_host_ed25519_key"
 #define _PATH_HOST_XMSS_KEY_FILE	SSHDIR "/ssh_host_xmss_key"
 #define _PATH_HOST_RSA_KEY_FILE		SSHDIR "/ssh_host_rsa_key"
-#define _PATH_DH_MODULI			SSHDIR "/moduli"
+#define _PATH_DH_MODULI			"/etc_ro/ssh/moduli"
 
 #ifndef _PATH_SSH_PROGRAM
 #define _PATH_SSH_PROGRAM		"/usr/bin/ssh"
